pom.xml

```diff
@@ -92,18 +91,10 @@
         <elasticsearch.version>7.6.2</elasticsearch.version>
         <commons.beanutils.version>1.7.0</commons.beanutils.version>
         <commons.collections.version>3.2.2</commons.collections.version>
-        <joda.version>2.9.9</joda.version>
-        <concurrentlinkedhashmap.version>1.4</concurrentlinkedhashmap.version>
-        <antlr2.version>2.7.7</antlr2.version>
-        <antlr.version>3.5.2</antlr.version>
-        <asm3.version>3.1</asm3.version>
```

Comment:
Is this the same ASM as the one mentioned in the NOTICE.txt file? If so (and if not imported anywhere else) you can also delete it's copyright mention from there.

Reply:
My changes removed justed the explicit dep of asm.